server/middleware/rateLimited.js

```diff
@@ -0,0 +1,11 @@
+const rateLimit = require('express-rate-limit');
+
+module.exports = {
+    rateLimiterUsingThirdParty: rateLimit({
+        windowMs: 24 * 60 * 60 * 1000, // 24 hrs in milliseconds
+        max: 100,
```

Comment:
Let's make 50 req max per day.

Reply:
Will be addressed here: https://github.com/TheRedCandyy/MassDiscoverWebsite/issues/36

Reply:
Lindooooo